File: javascript/rna-transcription/rna-transcription.js
var DnaTranscriber = function(){

}


DnaTranscriber.prototype = {
  toRna: function(string){
    var rna = ""
    string.forEach(character){
      switch(character){
        case "C": rna += "G";
        break;
        case "G": rna += "C";
        break;
        case "T": rna += "A";
        break;
        case "A": rna += "U";
      }

    }

  }

}

module.exports = DnaTranscriber;

File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):

    dna_list = list(dna_strand)

    rna_string = ""

    for character in dna_list:
        if character == "G":
            rna_string += "C"
        elif character == "C":
            rna_string += "G"
        elif character == "T":
            rna_string += "A"
        elif character == "A":
            rna_string += "U"

    return rna_string





